docs/application/web/guides/security/privacy-related-permissions.md

```diff
@@ -18,6 +18,11 @@ The main features of the Privacy Privilege API include:
 
 For a list of privacy-related privileges, see [Security and API Privileges](../../tutorials/sec-privileges.md).
 
+> [!NOTE]
```

Comment:
@yl33  Kindly refer to our [custom style guide](https://github.com/Samsung/tizen-docs/blob/master/styleguide/custom-style.md) to know more about new note styling.

We do not need an empty space after note tag, so:
> [!NOTE]
> Since Tizen 5.5, pop-ups by `requestPermission()` and `requestPermissions()` are launched as `group mode` with the caller apps.   
> If the pop-up is terminated without full response, all the remaining requests that aren't responded by the user will be interpreted as 'Deny' action on behalf of the user. In this case, the app gets `PPM_DENY_ONCE` response to those permissions. The app can again request for those **automatically denied** permissions anytime it needs.  
> User can use `requestPermissions()` to request for multiple privileges instead of calling `requestPermission()` multiple times.


@yl33 I slightly modified the sentence for more clarity, request you to verify the sentence technically and if you feel this is fine, kindly incorporate the changes

Reply:
> User can use `requestPermissions()` to request for multiple privileges instead of calling `requestPermission()` multiple times.

This note is not for the users but for the developers. Modified your comment slightly.
Modified other phrases as you mentioned.